i18n/locales/en.json

```diff
@@ -58,7 +58,6 @@
   "components.home.nodeFeatures.openSource.description": "Node.js is open source and actively maintained by contributors all over the world",
   "components.home.nodeFeatures.everywhere.title": "Everywhere",
   "components.home.nodeFeatures.everywhere.description": "Node.js has been adapted to work in a wide variety of places",
-  "components.common.shellBox.copy": "{copied, select, true {copied}other {copy}}",
   "components.codeBox.copy": "{copied, select, true {copied}other {copy}}",
```

Comment:
This can be removed as aria is getting gone. Also this should have ben `compoonents.common.codeBox.copy` 😢